client/src/components/WarmFuzziesIndex.js

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from "react";
+import WarmFuzzyClient from "../apiClient/WarmFuzzyClient";
+import WarmFuzzyItem from "../components/WarmFuzzyItem";
+import { useAuthentication } from "../providers/Authentication";
+
+const WarmFuzziesIndex = (props) => {
+  const { user } = useAuthentication();
+  const [warmFuzzies, setWarmFuzzies] = useState([]);
+
+  const fetchWarmFuzzies = () => {
+    const client = new WarmFuzzyClient()
+    const allWarmFuzzies = client.getAllFuzzies()
+      .then(resp => {
+        return resp;
+      })
+      .then(fuzzies => {
+        console.log("returned fuzzies here")
+        setWarmFuzzies(fuzzies)
+      })
+  }
+
+  useEffect(fetchWarmFuzzies, []);
```

Comment:
good. As an optimization here, we could make use of the `useCallback` function. 

```javascript
const fetchWarmFuzzies = useCallBack(() => {/* function body */}, [setWarmFuzzies])
```

Reply:
It would be helpful for me to talk through this.